CorePreferences/release/modules/ext/docs/CorePreferences/src/au/gov/asd/tac/constellation/preferences/application-preferences.md

```diff
@@ -50,8 +52,8 @@ Constellation via Tools -> Start Jupyter Notebook.
 
 Selecting to "Download Python REST client" means that when you start the
 Jupyter Notebook server, Constellation will download the
-constellation_client.py file to your .ipython folder. The
-constellation_client.py provides easy access to make Restful calls to
+constellation\_client.py file to your .ipython folder. The
```

Comment:
I think this sentence is technically incorrect now as it is now pip installing the client

Reply:
One way this could be reworded is:

Selecting to "Download Python REST client" means that when you start the Jupyter Notebook server, Constellation will install our custom python package, constellation_client.
The constellation_client package provides a convenient interface to the REST API for Python scripts in Jupyter notebooks.